src/naive/date/mod.rs

```diff
@@ -155,6 +157,11 @@ impl NaiveDate {
         ((self.yof & ORDINAL_MASK) >> 4) as u32
     }
 
+    #[inline]
+    const fn year_flags(&self) -> YearFlags {
```

Comment:
It's not a move, because it just appears?

Reply:
Yes, there was one more use of `Of::flags` that made me unable to remove its counterpart. I'll try to reorder the commits.